packages/firebase/index.d.ts

```diff
@@ -3961,7 +3961,21 @@ declare namespace firebase.analytics {
      * ```
      */
     app: firebase.app.App;
-
+              
+     /**
+     * Sends analytics event with given `eventParams`. This method
+     * automatically associates this logged event with this Firebase web
+     * app instance on this device.
+     * List of official event parameters can be found in
+     * {@link https://developers.google.com/gtagjs/reference/event
+     * the gtag.js reference documentation}.
+     */
+    logEvent(
+      eventName: 'add_payment_info',
+      eventParams?: {},
```

Comment:
In the case of `add_payment_info` it seems there's no specific recommended parameters, so can you just replace the `eventParams?: {}` with `eventParams?: { [key: string]: any }` so it will at least have an index signature?